### ISSUE TITLE
Windows Explorer Issue

### ISSUE THREAD

**Issue by Zetanova** [micromata/dave#37]
I tried now dave over docker local and over https (k8s nginx ingress)

WebDav clients like filestash online demo can connect without issues,
but Windows 10 Explorer is reporting an error after user-credentials

"The folder you entered does not appear to be valid. Please choose abother"

![image](https://user-images.githubusercontent.com/1385312/157292976-926434bc-acdc-4373-98e2-f9d3ab6064ee.png)

**Comment by mischah** [comment]
Is this really a dave issue? Is someone with Windows 10 able to reproduce this?
Are there any »unusal« characters used in your folder that might break windows 🤔

**Comment by Zetanova** [comment]
It was a blank usage of the dave container image. 
I didn't investigate more then this.

**Comment by mischah** [comment]
I don’t think that one of the maintainers will investigate more into this a long this is the only report of this kinda issue (and windows only) 🧐

Guess we should leave this open and wait for others to possible stumble on this (in case that’s a bug).

**Comment by samhocevar** [comment]
Hi! This may happen if the WebClient service is not running on your machine. Can you maybe check this?

![image](https://user-images.githubusercontent.com/245089/222398577-12324330-814b-4b28-a92f-1e4d0b68cffc.png)

**Comment by divinity76** [comment]
I also have a problem connecting with explorer.exe on windows 10, but i have a different issue with credentials, guess ill open a separate issue

**Comment by divinity76** [comment]
from what i can tell, there is a problem with how windows 10 handles authentication in OPTIONS requests. and the problem goes away if you disable authentication entirely (i.e delete all the users, or use https://github.com/micromata/dave/pull/50 to whitelist your ip)

**Comment by divinity76** [comment]
btw is anyone able to mount Dave with authentication in any windows version at all? i've only tried on win10, and i'm only able to do it with #50 (or with authentication completely disabled)
